diff --git a/data/ship/cfg/TR1X_gameflow.json5 b/data/ship/cfg/TR1X_gameflow.json5
index f15a010a..468aeb21 100644
--- a/data/ship/cfg/TR1X_gameflow.json5
+++ b/data/ship/cfg/TR1X_gameflow.json5
@@ -4,15 +4,15 @@
     // Refer to https://github.com/LostArtefacts/TR1X/blob/stable/GAMEFLOW.md
     // for usage.
 
-    "main_menu_picture": "data/images/title.webp",
+    "main_menu_picture": "pictures/title.png",
     "savegame_fmt_legacy": "saveati.%d",
     "savegame_fmt_bson": "save_tr1_%02d.dat",
-    "force_game_modes": null,
-    "force_save_crystals": null,
-    "demo_delay": 16,
+    "force_game_modes": false,
+    "force_save_crystals": false,
+    "demo_delay": 6,
     "water_color": [0.45, 1.0, 1.0],
-    "draw_distance_fade": 22.0,
-    "draw_distance_max": 30.0,
+    "draw_distance_fade": 14.0,
+    "draw_distance_max": 22.0,
     "injections": [
         "data/injections/backpack.bin",
         "data/injections/braid.bin",
@@ -23,554 +23,191 @@
     "convert_dropped_guns": false,
 
     "levels": [
-        // Level 0
-        {
-            "title": "Lara's Home",
-            "file": "data/gym.phd",
-            "type": "gym",
-            "music": 0,
-            "inherit_injections": false,
-            "injections": [
-                "data/injections/lara_gym_guns.bin",
-                "data/injections/braid.bin",
-                "data/injections/gym_textures.bin",
-                "data/injections/lara_animations.bin",
-                "data/injections/lara_jumping.bin",
-                "data/injections/uzi_sfx.bin",
-            ],
-            "sequence": [
-                {"type": "play_fmv", "fmv_path": "fmv/mansion.avi"},
-                {"type": "loading_screen", "picture_path": "data/images/gym.webp", "display_time": 5},
-                {"type": "start_game"},
-                {"type": "loop_game"},
-                {"type": "stop_game"},
-                {"type": "level_stats", "level_id": 0},
-                {"type": "exit_to_title"},
-            ],
-            "strings": {},
-        },
-
         // Level 1
         {
-            "title": "Caves",
-            "file": "data/level1.phd",
-            "type": "normal",
-            "music": 57,
-            "injections": [
-                "data/injections/caves_fd.bin",
-                "data/injections/caves_itemrots.bin",
-                "data/injections/caves_textures.bin",
-            ],
-            "sequence": [
-                {"type": "play_fmv", "fmv_path": "fmv/snow.avi"},
-                {"type": "loading_screen", "picture_path": "data/images/peru.webp", "display_time": 5},
-                {"type": "start_game"},
-                {"type": "loop_game"},
-                {"type": "stop_game"},
-                {"type": "level_stats", "level_id": 1},
-                {"type": "exit_to_level", "level_id": 2},
-            ],
-            "strings": {},
-        },
-
-        // Level 2
-        {
-            "title": "City of Vilcabamba",
-            "file": "data/level2.phd",
-            "type": "normal",
-            "music": 57,
-            "injections": [
-                "data/injections/vilcabamba_itemrots.bin",
-                "data/injections/vilcabamba_textures.bin",
-            ],
-            "sequence": [
-                {"type": "loading_screen", "picture_path": "data/images/peru.webp", "display_time": 5},
-                {"type": "start_game"},
-                {"type": "loop_game"},
-                {"type": "stop_game"},
-                {"type": "level_stats", "level_id": 2},
-                {"type": "exit_to_level", "level_id": 3},
-            ],
-            "strings": {
-                "key1": "Silver Key",
-                "puzzle1": "Gold Idol"
-            },
-            "demo": true,
-        },
-
-        // Level 3
-        {
-            "title": "Lost Valley",
-            "file": "data/level3a.phd",
+            "title": "Croft Manor",
+            "file": "data/Manor.phd",
             "type": "normal",
-            "music": 57,
-            "injections": [
-                "data/injections/valley_itemrots.bin",
-                "data/injections/valley_skybox.bin",
-                "data/injections/valley_textures.bin",
-            ],
+            "music": 60,
             "sequence": [
-                {"type": "loading_screen", "picture_path": "data/images/peru.webp", "display_time": 5},
+                {"type": "play_fmv", "fmv_path": "fmv/fmv1.avi"},
+                {"type": "remove_guns"},
                 {"type": "start_game"},
                 {"type": "loop_game"},
                 {"type": "stop_game"},
-                {"type": "level_stats", "level_id": 3},
-                {"type": "exit_to_level", "level_id": 4},
+                {"type": "level_stats", "level_id": 0},
+                {"type": "exit_to_level", "level_id": 1},
             ],
             "strings": {
-                "puzzle1": "Machine Cog"
+                "key1": "Basement Key",
+                "key2": "Closet Key",
+                "key3": "Storage Room Key",
+                "key4": "Tomb Key",
+                "puzzle1": "Thistle Stone",
+                "puzzle2": "Cogwheel",
+                "puzzle3": "Gold Bar"
             },
-            "demo": true,
         },
 
-        // Level 4
+        // Level 2
         {
-            "title": "Tomb of Qualopec",
-            "file": "data/level3b.phd",
+            "title": "Manor Underground",
+            "file": "data/Underground.phd",
             "type": "normal",
             "music": 57,
-            "injections": [
-                "data/injections/larson_textures.bin",
-                "data/injections/qualopec_fd.bin",
-                "data/injections/qualopec_itemrots.bin",
-                "data/injections/qualopec_textures.bin",
-            ],
             "sequence": [
-                {"type": "loading_screen", "picture_path": "data/images/peru.webp", "display_time": 5},
+                {"type": "fix_trapdoor"},
                 {"type": "start_game"},
                 {"type": "loop_game"},
                 {"type": "stop_game"},
-                {"type": "exit_to_cine", "level_id": 16},
-            ],
-            "strings": {},
-        },
-
-        // Level 5
-        {
-            "title": "St. Francis' Folly",
-            "file": "data/level4.phd",
-            "type": "normal",
-            "music": 59,
-            "injections": [
-                "data/injections/folly_fd.bin",
-                "data/injections/folly_itemrots.bin",
-                "data/injections/folly_textures.bin",
-            ],
-            "sequence": [
-                {"type": "play_fmv", "fmv_path": "fmv/lift.avi"},
-                {"type": "loading_screen", "picture_path": "data/images/greece.webp", "display_time": 5},
-                {"type": "start_game"},
-                {"type": "loop_game"},
-                {"type": "stop_game"},
-                {"type": "level_stats", "level_id": 5},
-                {"type": "exit_to_level", "level_id": 6},
-            ],
-            "strings": {
-                "key1": "Neptune Key",
-                "key2": "Atlas Key",
-                "key3": "Damocles Key",
-                "key4": "Thor Key"
-            },
-        },
-
-        // Level 6
-        {
-            "title": "Colosseum",
-            "file": "data/level5.phd",
-            "type": "normal",
-            "music": 59,
-            "injections": [
-                "data/injections/colosseum_door.bin",
-                "data/injections/colosseum_fd.bin",
-                "data/injections/colosseum_itemrots.bin",
-                "data/injections/colosseum_skybox.bin",
-                "data/injections/colosseum_textures.bin",
-            ],
-            "sequence": [
-                {"type": "loading_screen", "picture_path": "data/images/greece.webp", "display_time": 5},
-                {"type": "start_game"},
-                {"type": "loop_game"},
-                {"type": "stop_game"},
-                {"type": "level_stats", "level_id": 6},
-                {"type": "exit_to_level", "level_id": 7},
+                {"type": "level_stats", "level_id": 1},
+                {"type": "exit_to_level", "level_id": 2},
             ],
             "strings": {
-                "key1": "Rusty Key"
+                "key1": "Storeroom Key",
+                "key2": "Rusty Key",
+                "key4": "Detonator Key",
+                "puzzle1": "Fuse",
+                "puzzle2": "Circuit Board"
             },
         },
 
-        // Level 7
+        // Level 3
         {
-            "title": "Palace Midas",
-            "file": "data/level6.phd",
+            "title": "Dark Fortress",
+            "file": "data/Fortress.phd",
             "type": "normal",
             "music": 59,
-            "injections": [
-                "data/injections/midas_itemrots.bin",
-                "data/injections/midas_textures.bin",
-            ],
+            "water_color": [0.4, 0.4, 0.4],
             "sequence": [
-                {"type": "loading_screen", "picture_path": "data/images/greece.webp", "display_time": 5},
+                {"type": "play_fmv", "fmv_path": "fmv/fmv3.mp4"},
                 {"type": "start_game"},
                 {"type": "loop_game"},
                 {"type": "stop_game"},
-                {"type": "level_stats", "level_id": 7},
-                {"type": "exit_to_level", "level_id": 8},
-            ],
-            "strings": {
-                "puzzle1": "Gold Bar"
-            },
-            "unobtainable_pickups": 1,
-        },
-
-        // Level 8
-        {
-            "title": "The Cistern",
-            "file": "data/level7a.phd",
-            "type": "normal",
-            "music": 58,
-            "injections": [
-                "data/injections/cistern_fd.bin",
-                "data/injections/cistern_itemrots.bin",
-                "data/injections/cistern_plants.bin",
-                "data/injections/cistern_textures.bin",
-            ],
-            "sequence": [
-                {"type": "loading_screen", "picture_path": "data/images/greece.webp", "display_time": 5},
-                {"type": "start_game"},
-                {"type": "loop_game"},
-                {"type": "stop_game"},
-                {"type": "level_stats", "level_id": 8},
-                {"type": "exit_to_level", "level_id": 9},
+                {"type": "level_stats", "level_id": 2},
+                {"type": "exit_to_level", "level_id": 3},
             ],
             "strings": {
                 "key1": "Gold Key",
                 "key2": "Silver Key",
-                "key3": "Rusty Key"
+                "key3": "Dungeon Key",
+                "key4": "Underground Key",
+                "puzzle1": "Violet Gem",
+                "puzzle2": "Bucket of Black Water",
+                "puzzle3": "Knight's Diary"
             },
         },
 
-        // Level 9
+        // Level 4
         {
-            "title": "Tomb of Tihocan",
-            "file": "data/level7b.phd",
+            "title": "Chambers of Torture",
+            "file": "data/Chambers.phd",
             "type": "normal",
             "music": 58,
-            "injections": [
-                "data/injections/cistern_plants.bin",
-                "data/injections/tihocan_fd.bin",
-                "data/injections/tihocan_itemrots.bin",
-                "data/injections/tihocan_textures.bin",
-            ],
-            "item_drops": [
-                {"enemy_num": 82, "object_ids": [86, 144, 129]},
-            ],
-            "sequence": [
-                {"type": "loading_screen", "picture_path": "data/images/greece.webp", "display_time": 5},
+           "sequence": [
                 {"type": "start_game"},
                 {"type": "loop_game"},
                 {"type": "stop_game"},
-                {"type": "exit_to_cine", "level_id": 17},
+                {"type": "level_stats", "level_id": 3},
+                {"type": "exit_to_level", "level_id": 4},
             ],
             "strings": {
                 "key1": "Gold Key",
-                "key2": "Rusty Key",
-                "key3": "Rusty Key"
-            },
-        },
-
-        // Level 10
-        {
-            "title": "City of Khamoon",
-            "file": "data/level8a.phd",
-            "type": "normal",
-            "music": 59,
-            "injections": [
-                "data/injections/khamoon_fd.bin",
-                "data/injections/khamoon_mummy.bin",
-                "data/injections/khamoon_textures.bin",
-            ],
-            "sequence": [
-                {"type": "play_fmv", "fmv_path": "fmv/vision.avi"},
-                {"type": "loading_screen", "picture_path": "data/images/egypt.webp", "display_time": 5},
-                {"type": "start_game"},
-                {"type": "loop_game"},
-                {"type": "stop_game"},
-                {"type": "level_stats", "level_id": 10},
-                {"type": "exit_to_level", "level_id": 11},
-            ],
-            "strings": {
-                "key1": "Sapphire Key"
-            },
-        },
-
-        // Level 11
-        {
-            "title": "Obelisk of Khamoon",
-            "file": "data/level8b.phd",
-            "type": "normal",
-            "music": 59,
-            "injections": [
-                "data/injections/obelisk_fd.bin",
-                "data/injections/obelisk_itemrots.bin",
-                "data/injections/obelisk_skybox.bin",
-                "data/injections/obelisk_textures.bin",
-            ],
-            "sequence": [
-                {"type": "loading_screen", "picture_path": "data/images/egypt.webp", "display_time": 5},
-                {"type": "start_game"},
-                {"type": "loop_game"},
-                {"type": "stop_game"},
-                {"type": "level_stats", "level_id": 11},
-                {"type": "exit_to_level", "level_id": 12},
-            ],
-            "strings": {
-                "key1": "Sapphire Key",
-                "puzzle1": "Eye of Horus",
-                "puzzle2": "Scarab",
-                "puzzle3": "Seal of Anubis",
-                "puzzle4": "Ankh"
+                "key2": "Bronze Key",
+                "puzzle1": "Unholy Plaque",
+                "puzzle2": "Unknown's Diary",
+                "puzzle3": "North - 3, South - 1, West - 4, East - 2",
+                "puzzle4": "Use your compass to reveal the plaque. Remember, it ALWAYS points towards North!"
             },
         },
 
-        // Level 12
+        // Level 5
         {
-            "title": "Sanctuary of the Scion",
-            "file": "data/level8c.phd",
+            "title": "Black Palace",
+            "file": "data/Palace.phd",
             "type": "normal",
-            "music": 59,
-            "injections": [
-                "data/injections/sanctuary_fd.bin",
-                "data/injections/sanctuary_itemrots.bin",
-                "data/injections/sanctuary_textures.bin",
-            ],
+            "music": 56,
             "sequence": [
-                {"type": "loading_screen", "picture_path": "data/images/egypt.webp", "display_time": 5},
                 {"type": "start_game"},
+                {"type": "setup_bacon_lara", "anchor_room": 95},
                 {"type": "loop_game"},
                 {"type": "stop_game"},
-                {"type": "level_stats", "level_id": 12},
-                {"type": "exit_to_level", "level_id": 13},
+                {"type": "level_stats", "level_id": 4},
+                {"type": "exit_to_level", "level_id": 5},
             ],
             "strings": {
-                "key1": "Gold Key",
-                "puzzle1": "Ankh",
-                "puzzle2": "Scarab"
+                "key1": "Stone Key",
+                "key2": "Labyrinth Key",
+                "key3": "The Moon is born of black. Switch is to be used only once. Save the game.",
+                "key4": "Rooftop key",
+                "puzzle1": "Amethyst Bar",
+                "puzzle2": "Survivor's Diary",
+                "puzzle3": "Crowbar",
+                "puzzle4": "Knife"
             },
         },
 
-        // Level 13
+        // Level 6
         {
-            "title": "Natla's Mines",
-            "file": "data/level10a.phd",
+            "title": "Sanctuary of the Crystal",
+            "file": "data/Sanctuary.phd",
             "type": "normal",
             "music": 58,
-            "injections": [
-                "data/injections/cowboy_textures.bin",
-                "data/injections/kold_textures.bin",
-                "data/injections/mines_fd.bin",
-                "data/injections/mines_itemrots.bin",
-                "data/injections/mines_textures.bin",
-                "data/injections/skateboardkid_textures.bin"
-            ],
-            "item_drops": [
-                {"enemy_num": 17, "object_ids": [86]},
-                {"enemy_num": 50, "object_ids": [87]},
-                {"enemy_num": 75, "object_ids": [85]},
-            ],
+            "water_color": [0.4, 0.4, 0.4],
             "sequence": [
-                {"type": "play_fmv", "fmv_path": "fmv/canyon.avi"},
-                {"type": "loading_screen", "picture_path": "data/images/atlantis.webp", "display_time": 5},
-                {"type": "remove_guns"},
-                {"type": "remove_scions"},
                 {"type": "start_game"},
                 {"type": "loop_game"},
                 {"type": "stop_game"},
-                {"type": "level_stats", "level_id": 13},
-                {"type": "exit_to_cine", "level_id": 18},
+                {"type": "level_stats", "level_id": 5},
+                {"type": "play_fmv", "fmv_path": "fmv/fmv2.mp4"},
+                {"type": "play_synced_audio", "audio_id": 23},
+                {"type": "display_picture", "picture_path": "pictures/end1.png",
+"display_time": 15},
+                {"type": "display_picture", "picture_path": "pictures/end2.png",
+"display_time": 7.5},
+                {"type": "display_picture", "picture_path": "pictures/end3.png",
+"display_time": 8},
+                {"type": "display_picture", "picture_path": "pictures/end4.png",
+"display_time": 10},
+                {"type": "display_picture", "picture_path": "pictures/end5.png",
+"display_time": 10},
+                {"type": "display_picture", "picture_path": "pictures/end6.png",
+"display_time": 10},
+                {"type": "display_picture", "picture_path": "pictures/end7.png",
+"display_time": 10},
+                {"type": "display_picture", "picture_path": "pictures/end8.png",
+"display_time": 10},
+                {"type": "display_picture", "picture_path": "pictures/end12.png"
+,"display_time": 10},
+                {"type": "display_picture", "picture_path": "pictures/end11.png"
+,"display_time": 10},
+                {"type": "display_picture", "picture_path": "pictures/end9.png",
+"display_time": 15},
+                {"type": "total_stats", "picture_path": "pictures/end10.png"},
+                {"type": "exit_to_title"},
             ],
             "strings": {
-                "puzzle1": "Fuse",
-                "puzzle2": "Pyramid Key"
+                "puzzle1": "Dark Crystal",
+                "puzzle3": "Beware of him..."
             },
+            "unobtainable_pickups": 1,
         },
 
-        // Level 14
-        {
-            "title": "Atlantis",
-            "file": "data/level10b.phd",
-            "type": "normal",
-            "music": 60,
-            "injections": [
-                "data/injections/atlantis_fd.bin",
-                "data/injections/atlantis_textures.bin",
-            ],
-            "sequence": [
-                {"type": "play_fmv", "fmv_path": "fmv/pyramid.avi"},
-                {"type": "loading_screen", "picture_path": "data/images/atlantis.webp", "display_time": 5},
-                {"type": "start_game"},
-                {"type": "give_item", "object_id": 84, "quantity": 1},
-                {"type": "setup_bacon_lara", "anchor_room": 10},
-                {"type": "loop_game"},
-                {"type": "stop_game"},
-                {"type": "play_fmv", "fmv_path": "fmv/prison.avi"},
-                {"type": "exit_to_cine", "level_id": 19},
-            ],
-            "strings": {},
-            "unobtainable_pickups": 3,
-        },
-
-        // Level 15
-        {
-            "title": "The Great Pyramid",
-            "file": "data/level10c.phd",
-            "type": "normal",
-            "music": 60,
-            "injections": [
-                "data/injections/pyramid_fd.bin",
-                "data/injections/pyramid_textures.bin",
-                "data/injections/scion_collision.bin",
-            ],
-            "sequence": [
-                {"type": "loading_screen", "picture_path": "data/images/atlantis.webp", "display_time": 5},
-                {"type": "start_game"},
-                {"type": "loop_game"},
-                {"type": "stop_game"},
-                {"type": "level_stats", "level_id": 15},
-                {"type": "play_fmv", "fmv_path": "fmv/end.avi"},
-                {"type": "play_synced_audio", "audio_id": 19},
-                {"type": "display_picture", "picture_path": "data/images/end.webp", "display_time": 7.5},
-                {"type": "display_picture", "picture_path": "data/images/credits_1.webp", "display_time": 7.5},
-                {"type": "display_picture", "picture_path": "data/images/credits_2.webp", "display_time": 7.5},
-                {"type": "display_picture", "picture_path": "data/images/credits_3.webp", "display_time": 7.5},
-                {"type": "total_stats", "picture_path": "data/images/install.webp"},
-                {"type": "exit_to_title"},
-            ],
-            "strings": {},
-        },
-
-        // Level 16
-        {
-            "title": "Cut Scene 1",
-            "file": "data/cut1.phd",
-            "type": "cutscene",
-            "music": 0,
-            "lara_type": 77,
-            "inherit_injections": false,
-            "injections": [
-                "data/injections/backpack_cut.bin",
-                "data/injections/braid_cut1.bin"
-            ],
-            "sequence": [
-                {"type": "start_cine"},
-                {"type": "set_cam_x", "value": 36668},
-                {"type": "set_cam_z", "value": 63180},
-                {"type": "set_cam_angle", "value": -23312},
-                {"type": "play_synced_audio", "audio_id": 23},
-                {"type": "loop_cine"},
-                {"type": "level_stats", "level_id": 4},
-                {"type": "exit_to_level", "level_id": 5},
-            ],
-            "strings": {},
-        },
-
-        // Level 17
-        {
-            "title": "Cut Scene 2",
-            "file": "data/cut2.phd",
-            "type": "cutscene",
-            "music": 0,
-            "lara_type": 77,
-            "inherit_injections": false,
-            "injections": [
-                "data/injections/backpack_cut.bin",
-                "data/injections/braid_cut2_cut4.bin"
-            ],
-            "sequence": [
-                {"type": "start_cine"},
-                {"type": "set_cam_x", "value": 51962},
-                {"type": "set_cam_z", "value": 53760},
-                {"type": "set_cam_angle", "value": 16380},
-                {"type": "play_synced_audio", "audio_id": 25},
-                {"type": "mesh_swap", "object1_id": 77, "object2_id": 1, "mesh_id": 1},
-                {"type": "mesh_swap", "object1_id": 77, "object2_id": 1, "mesh_id": 4},
-                {"type": "loop_cine"},
-                {"type": "level_stats", "level_id": 9},
-                {"type": "exit_to_level", "level_id": 10},
-            ],
-            "strings": {},
-        },
-
-        // Level 18
-        {
-            "title": "Cut Scene 3",
-            "file": "data/cut3.phd",
-            "type": "cutscene",
-            "music": 0,
-            "inherit_injections": false,
-            "injections": [
-                "data/injections/cut3_textures.bin",
-            ],
-            "sequence": [
-                {"type": "start_cine"},
-                {"type": "flip_map"},
-                {"type": "set_cam_angle", "value": 16384},
-                {"type": "play_synced_audio", "audio_id": 24},
-                {"type": "loop_cine"},
-                {"type": "exit_to_level", "level_id": 14},
-            ],
-            "strings": {},
-        },
-
-        // Level 19
-        {
-            "title": "Cut Scene 4",
-            "file": "data/cut4.phd",
-            "type": "cutscene",
-            "music": 0,
-            "draw_distance_fade": 12.0,
-            "draw_distance_max": 18.0,
-            "lara_type": 77,
-            "inherit_injections": false,
-            "injections": [
-                "data/injections/backpack_cut.bin",
-                "data/injections/braid_cut2_cut4.bin",
-                "data/injections/cut4_textures.bin",
-            ],
-            "sequence": [
-                {"type": "start_cine"},
-                {"type": "set_cam_angle", "value": 16384},
-                {"type": "play_synced_audio", "audio_id": 22},
-                {"type": "mesh_swap", "object1_id": 77, "object2_id": 1, "mesh_id": 1},
-                {"type": "mesh_swap", "object1_id": 77, "object2_id": 1, "mesh_id": 4},
-                {"type": "loop_cine"},
-                {"type": "level_stats", "level_id": 14},
-                {"type": "exit_to_level", "level_id": 15},
-            ],
-            "strings": {},
-        },
-
-        // Level 20
+        // Level 7
         {
             "title": "Title",
             "file": "data/title.phd",
             "type": "title",
             "music": 2,
             "inherit_injections": false,
-            "injections": [
-                "data/injections/pda_model.bin",
-            ],
             "sequence": [
-                {"type": "display_picture", "picture_path": "data/images/eidos.webp", "display_time": 1},
-                {"type": "play_fmv", "fmv_path": "fmv/core.avi"},
-                {"type": "play_fmv", "fmv_path": "fmv/escape.avi"},
-                {"type": "play_fmv", "fmv_path": "fmv/cafe.avi"},
+                {"type": "display_picture", "picture_path": "pictures/intro.png", "display_time": 1},
                 {"type": "exit_to_title"},
             ],
             "strings": {},
         },
 
-        // Level 21
+        // Level 8
         {
             "title": "Current Position",
             "file": "data/current.phd",
diff --git a/src/config_map.def b/src/config_map.def
index c2e9c683..d2c92c43 100644
--- a/src/config_map.def
+++ b/src/config_map.def
@@ -9,39 +9,39 @@ CFG_BOOL(enable_enhanced_look, true)
 CFG_BOOL(enable_shotgun_flash, true)
 CFG_BOOL(fix_shotgun_targeting, true)
 CFG_BOOL(enable_cheats, false)
-CFG_BOOL(enable_console, true)
+CFG_BOOL(enable_console, false)
 CFG_BOOL(enable_numeric_keys, true)
 CFG_BOOL(enable_tr3_sidesteps, true)
-CFG_BOOL(enable_braid, true)
+CFG_BOOL(enable_braid, false)
 CFG_BOOL(enable_compass_stats, true)
 CFG_BOOL(enable_total_stats, true)
 CFG_BOOL(enable_timer_in_inventory, true)
-CFG_BOOL(enable_smooth_bars, true)
+CFG_BOOL(enable_smooth_bars, false)
 CFG_BOOL(enable_fade_effects, true)
-CFG_BOOL(fix_tihocan_secret_sound, true)
-CFG_BOOL(fix_floor_data_issues, true)
+CFG_BOOL(fix_tihocan_secret_sound, false)
+CFG_BOOL(fix_floor_data_issues, false)
 CFG_BOOL(fix_secrets_killing_music, true)
 CFG_BOOL(fix_speeches_killing_music, true)
-CFG_BOOL(fix_descending_glitch, false)
-CFG_BOOL(fix_wall_jump_glitch, false)
+CFG_BOOL(fix_descending_glitch, true)
+CFG_BOOL(fix_wall_jump_glitch, true)
 CFG_BOOL(fix_bridge_collision, true)
 CFG_BOOL(fix_qwop_glitch, false)
 CFG_BOOL(fix_alligator_ai, true)
-CFG_BOOL(change_pierre_spawn, true)
+CFG_BOOL(change_pierre_spawn, false)
 CFG_BOOL(fix_bear_ai, true)
 CFG_INT32(fov_value, 65)
 CFG_INT32(resolution_width, -1)
 CFG_INT32(resolution_height, -1)
 CFG_BOOL(fov_vertical, true)
-CFG_BOOL(enable_demo, true)
+CFG_BOOL(enable_demo, false)
 CFG_BOOL(enable_fmv, true)
 CFG_BOOL(enable_eidos_logo, true)
 CFG_BOOL(enable_loading_screens, false)
-CFG_BOOL(enable_cine, true)
+CFG_BOOL(enable_cine, false)
 CFG_BOOL(enable_music_in_menu, true)
 CFG_BOOL(enable_music_in_inventory, true)
 CFG_BOOL(enable_round_shadow, true)
-CFG_BOOL(enable_3d_pickups, true)
+CFG_BOOL(enable_3d_pickups, false)
 CFG_FLOAT(rendering.anisotropy_filter, 16.0f)
 CFG_INT32(rendering.turbo_speed, 0)
 CFG_BOOL(walk_to_items, false)
@@ -52,29 +52,29 @@ CFG_ENUM(airbar_show_mode, BSM_DEFAULT, BAR_SHOW_MODE)
 CFG_ENUM(enemy_healthbar_show_mode, BSM_ALWAYS, BAR_SHOW_MODE)
 CFG_ENUM(healthbar_location, BL_TOP_LEFT, BAR_LOCATION)
 CFG_ENUM(airbar_location, BL_TOP_RIGHT, BAR_LOCATION)
-CFG_ENUM(enemy_healthbar_location, BL_BOTTOM_LEFT, BAR_LOCATION)
-CFG_ENUM(healthbar_color, BC_RED, BAR_COLOR)
+CFG_ENUM(enemy_healthbar_location, BL_BOTTOM_CENTER, BAR_LOCATION)
+CFG_ENUM(healthbar_color, BC_GOLD, BAR_COLOR)
 CFG_ENUM(airbar_color, BC_BLUE, BAR_COLOR)
-CFG_ENUM(enemy_healthbar_color, BC_GREY, BAR_COLOR)
-CFG_ENUM(screenshot_format, SCREENSHOT_FORMAT_JPEG, SCREENSHOT_FORMAT)
+CFG_ENUM(enemy_healthbar_color, BC_RED, BAR_COLOR)
+CFG_ENUM(screenshot_format, SCREENSHOT_FORMAT_PNG, SCREENSHOT_FORMAT)
 CFG_ENUM(ui.menu_style, UI_STYLE_PC, UI_STYLE)
 CFG_ENUM(target_mode, TLM_FULL, TARGET_LOCK_MODE)
 CFG_INT32(maximum_save_slots, 25)
-CFG_BOOL(revert_to_pistols, false)
+CFG_BOOL(revert_to_pistols, true)
 CFG_BOOL(enable_enhanced_saves, true)
 CFG_BOOL(enable_pitched_sounds, true)
 CFG_BOOL(enable_ps_uzi_sfx, false)
-CFG_BOOL(enable_jump_twists, true)
+CFG_BOOL(enable_jump_twists, false)
 CFG_BOOL(enabled_inverted_look, false)
 CFG_INT32(camera_speed, 5)
-CFG_BOOL(fix_texture_issues, true)
+CFG_BOOL(fix_texture_issues, false)
 CFG_BOOL(enable_swing_cancel, true)
-CFG_BOOL(enable_tr2_jumping, false)
+CFG_BOOL(enable_tr2_jumping, true)
 CFG_BOOL(enable_tr2_swimming, false)
 CFG_BOOL(load_current_music, true)
 CFG_BOOL(load_music_triggers, true)
 CFG_BOOL(fix_item_rots, true)
 CFG_BOOL(restore_ps1_enemies, false)
-CFG_BOOL(enable_game_modes, true)
+CFG_BOOL(enable_game_modes, false)
 CFG_BOOL(enable_save_crystals, false)
 CFG_BOOL(enable_uw_roll, true)
 CFG_BOOL(enable_buffering, false)
@@ -88,15 +88,15 @@ CFG_INT32(rendering.window_x, -1)
 CFG_INT32(rendering.window_y, -1)
 CFG_INT32(rendering.window_width, -1)
 CFG_INT32(rendering.window_height, -1)
-CFG_INT32(rendering.fps, 30)
+CFG_INT32(rendering.fps, 60)
 CFG_INT32(rendering.texture_filter, GFX_TF_BILINEAR)
 CFG_INT32(rendering.fbo_filter, GFX_TF_NN)
 CFG_BOOL(rendering.enable_wireframe, false)
 CFG_DOUBLE(rendering.wireframe_width, 2.5)
 CFG_BOOL(rendering.enable_perspective_filter, true)
 CFG_BOOL(rendering.enable_vsync, true)
-CFG_INT32(music_volume, 8)
-CFG_INT32(sound_volume, 8)
+CFG_INT32(music_volume, 10)
+CFG_INT32(sound_volume, 7)
 CFG_INT32(input.layout, 0)
 CFG_INT32(input.cntlr_layout, 0)
 CFG_FLOAT(brightness, DEFAULT_BRIGHTNESS)
diff --git a/src/game/gameflow.c b/src/game/gameflow.c
index 026e846c..5bc83eec 100644
--- a/src/game/gameflow.c
+++ b/src/game/gameflow.c
@@ -9,6 +9,8 @@
 #include "game/lara.h"
 #include "game/music.h"
 #include "game/objects/creatures/bacon_lara.h"
+#include "game/objects/traps/rolling_ball.h"
+#include "game/objects/general/trapdoor.h"
 #include "game/phase/phase.h"
 #include "game/phase/phase_cutscene.h"
 #include "game/phase/phase_picture.h"
@@ -89,6 +91,7 @@ static const STRING_TO_ENUM_TYPE m_GameflowSeqTypeEnumMap[] = {
     { "set_cam_z", GFS_SET_CAM_Z },
     { "set_cam_angle", GFS_SET_CAM_ANGLE },
     { "flip_map", GFS_FLIP_MAP },
+    { "fix_trapdoor", GFS_FIX_TRAPDOOR },
     { "remove_guns", GFS_REMOVE_GUNS },
     { "remove_scions", GFS_REMOVE_SCIONS },
     { "remove_ammo", GFS_REMOVE_AMMO },
@@ -406,6 +409,8 @@ static bool GameFlow_LoadLevelSequence(
             break;
         }
         case GFS_FLIP_MAP:
+        case GFS_FIX_TRAPDOOR:
+            break;
         case GFS_REMOVE_GUNS:
         case GFS_REMOVE_SCIONS:
         case GFS_REMOVE_AMMO:
@@ -990,6 +995,7 @@ void GameFlow_Shutdown(void)
                     case GFS_SET_CAM_Z:
                     case GFS_SET_CAM_ANGLE:
                     case GFS_FLIP_MAP:
+                    case GFS_FIX_TRAPDOOR:
                     case GFS_REMOVE_GUNS:
                     case GFS_REMOVE_SCIONS:
                     case GFS_PLAY_SYNCED_AUDIO:
@@ -1248,6 +1254,11 @@ GameFlow_InterpretSequence(int32_t level_num, GAMEFLOW_LEVEL_TYPE level_type)
         case GFS_FLIP_MAP:
             Room_FlipMap();
             break;
+        case GFS_FIX_TRAPDOOR:
+                LOG_INFO("Load timed trapdoor fix");
+                g_RollingBallFix = 1;
+                g_TrapDoorFix = 1;
+            break;
         case GFS_PLAY_SYNCED_AUDIO:
             Music_Play((int32_t)(intptr_t)seq->data);
             break;
@@ -1433,6 +1444,8 @@ GameFlow_StorySoFar(int32_t level_num, int32_t savegame_level)
         case GFS_FLIP_MAP:
             Room_FlipMap();
             break;
+        case GFS_FIX_TRAPDOOR:
+            break;
         case GFS_PLAY_SYNCED_AUDIO:
             Music_Play((int32_t)(intptr_t)seq->data);
             break;
diff --git a/src/game/objects/general/trapdoor.c b/src/game/objects/general/trapdoor.c
index 0021839f..148bcb34 100644
--- a/src/game/objects/general/trapdoor.c
+++ b/src/game/objects/general/trapdoor.c
@@ -5,6 +5,8 @@
 
 #include <stdbool.h>
 
+int g_TrapDoorFix = 0;
+
 static bool TrapDoor_StandingOn(ITEM_INFO *item, int32_t x, int32_t z);
 
 static bool TrapDoor_StandingOn(ITEM_INFO *item, int32_t x, int32_t z)
@@ -34,12 +36,28 @@ void TrapDoor_Setup(OBJECT_INFO *obj)
     obj->save_flags = 1;
 }
 
+int testCounter = 50;
+
 void TrapDoor_Control(int16_t item_num)
 {
     ITEM_INFO *item = &g_Items[item_num];
     if (Item_IsTriggerActive(item)) {
         if (item->current_anim_state == DOOR_CLOSED) {
-            item->goal_anim_state = DOOR_OPEN;
+            if(g_TrapDoorFix && (item_num == 12))
+            {
+                if(testCounter)
+                {
+                    item->goal_anim_state = DOOR_CLOSED;
+                    testCounter -=1;
+                }
+                else
+                {
+                    item->goal_anim_state = DOOR_OPEN;
+                    testCounter = 50;
+                }
+            }
+            else
+                item->goal_anim_state = DOOR_OPEN;
         }
     } else if (item->current_anim_state == DOOR_OPEN) {
         item->goal_anim_state = DOOR_CLOSED;
diff --git a/src/game/objects/general/trapdoor.h b/src/game/objects/general/trapdoor.h
index ac38b3dc..531290b3 100644
--- a/src/game/objects/general/trapdoor.h
+++ b/src/game/objects/general/trapdoor.h
@@ -4,6 +4,8 @@
 
 #include <stdint.h>
 
+extern int g_TrapDoorFix;
+
 void TrapDoor_Setup(OBJECT_INFO *obj);
 void TrapDoor_Control(int16_t item_num);
 void TrapDoor_Floor(
diff --git a/src/game/objects/traps/rolling_ball.c b/src/game/objects/traps/rolling_ball.c
index 7b42b8e1..1308c10b 100644
--- a/src/game/objects/traps/rolling_ball.c
+++ b/src/game/objects/traps/rolling_ball.c
@@ -18,6 +18,8 @@
 
 #define ROLLINGBALL_DAMAGE_AIR 100
 
+int g_RollingBallFix = 0;
+
 void RollingBall_Setup(OBJECT_INFO *obj)
 {
     obj->initialise = RollingBall_Initialise;
@@ -42,6 +44,12 @@ void RollingBall_Initialise(int16_t item_num)
 void RollingBall_Control(int16_t item_num)
 {
     ITEM_INFO *item = &g_Items[item_num];
+    if(g_RollingBallFix && (item_num == 23) && (item->timer == -1))
+    {
+        g_Items[10].timer = -1;
+        g_Items[12].timer = -1;
+        g_Items[17].timer = -1;
+    }
     if (item->status == IS_ACTIVE) {
         if (item->pos.y < item->floor) {
             if (!item->gravity_status) {
diff --git a/src/game/objects/traps/rolling_ball.h b/src/game/objects/traps/rolling_ball.h
index 5db84d57..60982153 100644
--- a/src/game/objects/traps/rolling_ball.h
+++ b/src/game/objects/traps/rolling_ball.h
@@ -3,7 +3,7 @@
 #include "global/types.h"
 
 #include <stdint.h>
-
+extern int g_RollingBallFix;
 void RollingBall_Setup(OBJECT_INFO *obj);
 void RollingBall_Initialise(int16_t item_num);
 void RollingBall_Control(int16_t item_num);
diff --git a/src/game/option/option_passport.c b/src/game/option/option_passport.c
index fb3f8619..320eeafb 100644
--- a/src/game/option/option_passport.c
+++ b/src/game/option/option_passport.c
@@ -324,13 +324,13 @@ static void Option_PassportInitSelectLevelRequester(void)
 static void Option_PassportInitNewGameRequester(void)
 {
     REQUEST_INFO *req = &m_NewGameRequester;
-    Requester_ClearTextstrings(req);
-    Requester_SetHeading(req, GS(PASSPORT_SELECT_MODE));
-    Requester_AddItem(req, false, "%s", GS(PASSPORT_MODE_NEW_GAME));
-    Requester_AddItem(req, false, "%s", GS(PASSPORT_MODE_NEW_GAME_PLUS));
-    Requester_AddItem(req, false, "%s", GS(PASSPORT_MODE_NEW_GAME_JP));
-    Requester_AddItem(req, false, "%s", GS(PASSPORT_MODE_NEW_GAME_JP_PLUS));
-    req->vis_lines = MAX_GAME_MODES;
+    //Requester_ClearTextstrings(req);
+    //Requester_SetHeading(req, GS(PASSPORT_SELECT_MODE));
+    //Requester_AddItem(req, false, "%s", GS(PASSPORT_MODE_NEW_GAME));
+    //Requester_AddItem(req, false, "%s", GS(PASSPORT_MODE_NEW_GAME_PLUS));
+    //Requester_AddItem(req, false, "%s", GS(PASSPORT_MODE_NEW_GAME_JP));
+    //Requester_AddItem(req, false, "%s", GS(PASSPORT_MODE_NEW_GAME_JP_PLUS));
+    req->vis_lines = 0; //MAX_GAME_MODES;
 
     req->line_offset = 0;
     req->requested = 0;
diff --git a/src/global/const.h b/src/global/const.h
index b77761bd..02409f5e 100644
--- a/src/global/const.h
+++ b/src/global/const.h
@@ -17,10 +17,10 @@
 #define DEFAULT_UI_SCALE 1.0
 
 #define MIN_BRIGHTNESS 0.1f
-#define MAX_BRIGHTNESS 2.0f
+#define MAX_BRIGHTNESS 1.0f
 #define DEFAULT_BRIGHTNESS 1.0f
 
-#define LOGIC_FPS 30
+#define LOGIC_FPS 60
 
 #define MAX_MATRICES 40
 #define MAX_NESTED_MATRICES 32
diff --git a/src/global/types.h b/src/global/types.h
index 7586ef7e..9d9d975d 100644
--- a/src/global/types.h
+++ b/src/global/types.h
@@ -998,6 +998,7 @@ typedef enum GAMEFLOW_SEQUENCE_TYPE {
     GFS_SET_CAM_Z,
     GFS_SET_CAM_ANGLE,
     GFS_FLIP_MAP,
+    GFS_FIX_TRAPDOOR,
     GFS_REMOVE_GUNS,
     GFS_REMOVE_SCIONS,
     GFS_GIVE_ITEM,
